parsl/serialize/concretes.py

```diff
@@ -0,0 +1,155 @@
+import codecs
+import json
+import dill
+import pickle
+import inspect
+import logging
+
+logger = logging.getLogger(__name__)
+from parsl.serialize.base import fxPicker_shared
+
+
+class pickle_base64(fxPicker_shared):
+
+    _identifier = '10\n'
+    _for_code = False
+
+    def __init__(self):
+        super().__init__()
+
+    def serialize(self, data):
+        x = codecs.encode(pickle.dumps(data), 'base64').decode()
+        return self.identifier + x
+
+    def deserialize(self, payload):
+        chomped = self.chomp(payload)
+        data = pickle.loads(codecs.decode(chomped.encode(), 'base64'))
+        return data
+
+
+class dill_base64(fxPicker_shared):
+
+    _identifier = '11\n'
+    _for_code = False
+
+    def __init__(self):
+        super().__init__()
+
+    def serialize(self, data):
+        x = codecs.encode(dill.dumps(data), 'base64').decode()
+        return self.identifier + x
+
+    def deserialize(self, payload):
+        chomped = self.chomp(payload)
+        data = dill.loads(codecs.decode(chomped.encode(), 'base64'))
+        return data
+
+
+class json_base64(fxPicker_shared):
+    """ Json based serialization does not guarantee consistency on tuple/list types
```

Comment:
what do you mean by "consistency" here?

Reply:
Json serializer will convert tuples to lists.

Reply:
Ben, I've updated the docstrings on the serializers including json.